CMakeLists.txt

```diff
@@ -86,13 +104,42 @@ add_library(${LIB_TARGET_NAME} ${LIB_LINK_TYPE}
 string(TOUPPER "${CMAKE_BUILD_TYPE}" BUILD_CFG_NAME)
 set(CURRENT_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${CMAKE_CXX_FLAGS_${BUILD_CFG_NAME}}")
 
+set(JINJA2CPP_PUBLIC_LIBS "${JINJA2CPP_PUBLIC_LIBS}")
```

Comment:
Are quotes really necessary here? If `JINJA2CPP_PUBLIC_LIBS` is already a list, it will convert it to string.

Reply:
Lower I append to it using string concat so it is valuable